packages/nextjs/components/screens/Main.tsx

```diff
@@ -186,122 +186,127 @@ export const Main = () => {
-              handleSignature={handleSignature}
-            >
-              {loadingCheckedIn ? "..." : checkedIn ? "Checked-in" : "Check-in"}
-            </BurnerSigner>
-          </div>
-        )}
-
-        {checkedIn && !showBuy && !showSell && (
-          <>
-            <div className="bg-base-300 rounded-xl">
-              <table className="table-auto border-separate border-spacing-4">
-                <thead>
-                  <tr>
-                    <th>Token</th>
-                    <th>Price</th>
-                    <th>Balance</th>
-                    <th>Value</th>
-                    <th></th>
-                  </tr>
-                </thead>
-                <tbody>
-                  {tokens.map(token => (
-                    <TokenBalanceRow
-                      key={token.name}
-                      tokenInfo={token}
-                      tokenBalance={tokensData[token.name]}
-                      handleShowBuy={handleShowBuy}
-                      handleShowSell={handleShowSell}
-                      loading={loadingTokensData}
-                    />
-                  ))}
-                </tbody>
-              </table>
+              <h2>Sign Up to Play the Game</h2>
+              <div>
+                <InputBase
+                  value={alias}
+                  onChange={v => {
+                    setAlias(v);
+                  }}
+                  placeholder={alias ? alias : "Username"}
+                />
+              </div>
+
+              <BurnerSigner
+                className={`btn btn-primary w-full mt-4 ${processing || loadingCheckedIn ? "loading" : ""}`}
+                disabled={processing || loadingCheckedIn || checkedIn}
+                message={message}
+                handleSignature={handleSignature}
+              >
+                {loadingCheckedIn ? "..." : checkedIn ? "Checked-in" : "Check-in"}
+              </BurnerSigner>
             </div>
-
-            {scaffoldConfig.showChart && (
-              <>
-                <div className="flex gap-4 text-3xl mt-8">
-                  {tokens.map(token => (
-                    <label
-                      key={token.name}
-                      className={`p-2 cursor-pointer ${
-                        selectedTokenName === token.name ? "bg-primary outline outline-2 outline-black" : ""
-                      }`}
-                    >
-                      <input
-                        type="radio"
-                        name="token"
-                        value={token.name}
-                        className="w-0 h-0"
-                        onChange={t => setSelectedTokenName(t.target.value)}
+          )}
+
+          {checkedIn && !showBuy && !showSell && (
+            <>
+              <div className="rounded-xl overflow-x-scroll w-[350px]">
+                <table className="table-auto text-[0.8rem] border-seperate">
+                  <thead className="bg-[#F5F5F5]">
```

Comment:
Maybe we can put this color on tailwind config file https://github.com/BuidlGuidl/event-wallet/blob/main/packages/nextjs/tailwind.config.js

Reply:
Yeah I think it’s better I move that to the config